Mail/Views/Menu Drawer/Items/MenuDrawerItemCell.swift

```diff
@@ -45,19 +45,21 @@ struct MenuDrawerItemCell: View {
 
                 Text(label)
                     .textStyle(.bodyMedium)
+                    .multilineTextAlignment(.leading)
```

Comment:
Isn't .leading the default value?

Reply:
Maybe because we are inside a button label I had to specify it